src/Templates/AdminPanel/Bit.AdminPanel/src/Shared/Dtos/Account/EditUserDto.cs

```diff
@@ -13,13 +13,19 @@ public class EditUserDto
     [Display(Name = nameof(AppStrings.BirthDate))]
     public DateTimeOffset? BirthDate { get; set; }
 
-    public static implicit operator EditUserDto(UserDto user)
+    [JsonIgnore]
+    public string? GenderAsString
     {
-        return new()
+        get
         {
-            BirthDate = user.BirthDate,
-            FullName = user.FullName,
-            Gender = user.Gender
-        };
+            return Gender?.ToString();
+        }
+        set
+        {
+            if (string.IsNullOrEmpty(value) is false)
+            {
+                Gender = Enum.Parse<Gender>(value);
+            }
```

Comment:
```cs
if (string.IsNullOrEmpty(value) is false)
   Gender = Enum.Parse<Gender>(value);
else
   Gender = null; // what about else?
```

Reply:
it's an old code, I just moved it from UserDto. not sure its a valid chagne